backend/dal/Helpers/Extensions/BuildingExtensions.cs

```diff
@@ -24,11 +24,11 @@ public static class BuildingExtensions
         /// <exception type="DbUpdateException">The name within a parcel should be unique.</exception>
         public static void ThrowIfNotUnique(this PimsContext context, Entity.Parcel parcel, Entity.Building building)
         {
-            if(parcel == null)
+            if(parcel == null || String.IsNullOrEmpty(building.Name))
```

Comment:
Are you making the `name` a required field now?  Not sure the frontend has the same expectation.  Additionally this will most likely cause the import to fail as many buildings don't have names in the SIS data.

Reply:
I think this may have the opposite effect that you are thinking it does. If the building name is null or empty all of this duplicate name checking will be skipped.